var/spack/repos/builtin/packages/navi/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Navi(MakefilePackage):
```

Comment:
Since there isn't a makefile associated with the package, I think you want to inherit from `Package` here.

```suggestion
class Navi(Package):
```